charts/cloudzero-agent/templates/aggregator-deploy.yaml

```diff
@@ -0,0 +1,172 @@
+{{- if .Values.aggregator.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+{{- if .Values.aggregator.deploymentAnnotations }}
+  annotations:
+    {{- toYaml .Values.aggregator.deploymentAnnotations | nindent 4 }}
+{{- end }}
+  name: {{ include "cloudzero-agent.aggregator.name" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "cloudzero-agent.aggregator.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "cloudzero-agent.aggregator.matchLabels" . | nindent 6 }}
+  replicas: 1
```

Comment:
is this something we might want to scale horizontally? I guess in the case where a collector pod is rescheduled, it's okay because the prometheus agent will just try again if it gets a 503? so in that case 1 replica is okay?